pkg/assets/rte/assets.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 const (
-	OCPVersion411 = "4.11"
+	OCPVersion411 = "v4.11"
```

Comment:
could you please elaborate in the commit message why this is needed? If this makes some test fail, let's try to squash this commit in the one introducing the change (which then broke)

Reply:
We are using [ParseVersion](https://github.com/k8stopologyawareschedwg/deployer/blob/main/pkg/commands/root.go#L121) to parse the version which inturn calls [goversion.Parse](https://github.com/k8stopologyawareschedwg/deployer/blob/main/pkg/deployer/platform/version.go#L26) that checks if the version string matches the [regex](https://github.com/k8stopologyawareschedwg/deployer/blob/main/vendor/github.com/aquasecurity/go-version/pkg/version/version.go#L22-L25). Therefore we need to prefix version with the letter `v`. Also I noticed that in the example where we ask for platform explicitly we have an [example](https://github.com/k8stopologyawareschedwg/deployer/blob/main/pkg/commands/root.go#L93) that satisfies above.

Reply:
I think it is best to squash this commit as well.